docs/resources/file.md.erb

```diff
@@ -119,6 +119,22 @@ not
 
 Without the zero prefix for the octal value, InSpec will interpret it as the _decimal_ value 644, which is octal 1024 or `-----w-r-T`, and any test for a file that is `-rw-r--r--` will fail.
 
+### `more_permissive_than?(max_mode[String]) [Boolean]
+
+`more_permissive_than?` takes the maximum desired mode of your file as a `String` and returns a `Boolean`. It will return `true` if your file has a mode with greater permissions than specified.
+
+    describe file('/etc/passwd') do
+      it { should_not be_more_permissive_than?('0644') }
+      it { shold be_more_permissive_than?('0000') }
+    end
+
+    my_file = file('/etc/passwd')
+    describe "The file #{my_file.path} with permissions: `#{my_file.mode.to_s(8)}`" do
+      subject { my_file }
```

Comment:
We generally don't document the `subject` syntax.

Reply:
resolved in other PR